src/main/java/org/kainos/ea/utility/DatabaseConnector.java

```diff
@@ -0,0 +1,42 @@
+package org.kainos.ea.utility;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+import org.kainos.ea.exceptions.DatabaseConnectionException;
+
+public class DatabaseConnector {
+
+  private static Connection conn;
+  private static DbCredentials dbCredentials;
```

Comment:
This is used below but I can't see where we instantiate it. Do we need to include a `new DbCredentials()` call somewhere?

Reply:
Is it best practice to instantiate inside the try? I've updated the code like so:
<img width="775" alt="image" src="https://github.com/leeservice/as-team1-api/assets/137312132/1bdfcc67-f868-44f8-8634-86afba8ee457">

Alternatively I could instantiate dbCredentials with the local variables
<img width="762" alt="image" src="https://github.com/leeservice/as-team1-api/assets/137312132/d3f978a6-f187-4b0e-8163-fafe0e82eea6">

Reply:
I personally don't have a strong preference either way

Reply:
Updated and removed the static keyword
<img width="694" alt="image" src="https://github.com/leeservice/as-team1-api/assets/137312132/37aa3c3a-c66c-4090-8189-fa07a6b89743">